src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -636,6 +650,11 @@ String getAppPath(String appName) {
         return Collections.emptyList();
     }
 
+    // by default, false, but subclasses can decide against it.
```

Comment:
What is this method supposed to do/return?

Reply:
I've updated the javadoc